### DISCUSSION TITLE
If a name is in the simple registry is it taken or available?

### DISCUSSION THREAD

**Discussion by furminator123** [Stephen-RA-King/pynamer#16] [Q&A] [answered]
If a name does not show in the project area or on the search site is it available?

**Comment by Stephen-RA-King** [comment, marked as answer]
In this situation pynamer will give you the following message:

"A Gotcha!, whereby the package is not found even with PyPI's own search"
            " facility.\n"
            "It can only be found by searching the simple index which is not available "
            "through the web interface.\n\n"
            "The most likely cause is an abandoned or deleted project by the owner.\n\n"
            "Refer to PEP 541 – 'Package Index Name Retention' for details pertaining "
            "to name transfer

It may be available. It is worrth trying o register the name.  If it fails then try the PEP541 route.
